modules/tinymce/src/plugins/autoresize/main/ts/core/Resize.ts

```diff
@@ -29,6 +29,12 @@ const toggleScrolling = (editor: Editor, state: boolean): void => {
   }
 };
 
+interface ResizeData {
+  readonly totalHeight: number;
+  readonly contentHeight: number;
+  readonly initiated: boolean;
```

Comment:
This seems poorly named. If it's a flag that the height has been set on the editor already, how about calling it `forced` or `set`?

- [ ] consider renaming

Reply:
Something like that, which is why I thought `initated` would be a descriptive name. But if you prefer I can rename it, though both 'forced' and `set` also seem a bit lacking in detail as to what it refers to ( and forced would invert the current values, wouldn't it? ).